### DISCUSSION TITLE
Sending a ControlChange command

### DISCUSSION THREAD

**Discussion by abunezek** [melanchall/drywetmidi#219] [Q&A] [answered]
How do i simply send a ControlChange command into a n OutputDevice?
I need to send a value into a ch number and control number.

`outputDevice.SendEvent(new ControlChangeEvent());`

Thanks

**Comment by melanchall** [comment, marked as answer]
```csharp
outputDevice.SendEvent(new ControlChangeEvent(
    (SevenBitNumber)70, /* control number */
    (SevenBitNumber)50 /* control value */));
```

**Comment by abunezek** [comment]
Thanks!
I'm getting this error:
![image](https://user-images.githubusercontent.com/9842101/190653565-61083229-c201-4ffb-b0ec-4872058d21b3.png)
![image](https://user-images.githubusercontent.com/9842101/190653708-7dd97612-a98b-43cd-9ccc-1e87318daa87.png)
Do i need to declare this "sevenbitnumber"?

**Reply by melanchall** [reply]
You need to add using in the file

```csharp
using Melanchall.DryWetMidi.Common;
```

**Reply by abunezek** [reply]
Great!
And how do i set the channel?

**Reply by melanchall** [reply]
```csharp
outputDevice.SendEvent(new ControlChangeEvent((SevenBitNumber)70, (SevenBitNumber)50)
{
    Channel = (FourBitNumber)6
});
```

Please use the library documentation: https://melanchall.github.io/drywetmidi/api/Melanchall.DryWetMidi.Core.ControlChangeEvent.html

**Reply by abunezek** [reply]
Thanks a lot!
